src/components/footer/_footer.scss

```diff
@@ -30,12 +30,10 @@
     vertical-align: middle;
   }
 
-  &__poweredBy-link {
-    &:focus {
-      .ons-icon--logo {
-        @extend %a-focus;
-      }
-    }
+  &__extra-poweredby svg,
+  img {
+    height: 30px !important;
```

Comment:
Dina approved the size

Reply:
This would cause an issue where the ons logo would be bigger when there is 2 logos and smaller when there is just the ons logo so it wouldn't be consistent in size between pages